src/main/kotlin/no/nav/tiltaksarrangor/client/dto/GjennomforingDto.kt

```diff
@@ -13,6 +13,6 @@ data class GjennomforingDto(
 	val arrangor: ArrangorDto
 ) {
 	enum class Status {
-		IKKE_STARTET, APENT_FOR_INNSOK, GJENNOMFORES, AVSLUTTET
```

Comment:
Jan kanskje fjerne IKKE_STARTET fra Deltakerliste-statusen også? For den vil vel ikke være i bruk mer?

Reply:
Nå er jeg ikke sikker på hvor du mener. Søker etter IKKE_STARTET i repoet men finner ikke noen flere steder. Ser en deltakerlisteDto men denne har ikke status

Reply:
Fant den, søkte visst i feil repo